spec/polaris-management-service.yml

```diff
@@ -1056,6 +1056,13 @@ components:
               type: string
               description: endpoint for STS requests (optional). If not set, defaults to 'endpoint'.
               example: "https://sts.example.com:1234"
+            pathStyleAccess:
+              type: boolean
+              description: >- 
+                Whether S3 requests to files in this catalog should use 'path-style addressing for buckets'.
+                Default: false.
```

Comment:
nit: With the default in the spec, I don't think we need the default in the doc. Other similar variables don't have it.

Reply:
removed :+1: